pom.xml

```diff
@@ -10,12 +10,12 @@
         <groupId>org.alfresco</groupId>
         <artifactId>alfresco-enterprise-repo</artifactId>
         <relativePath>../alfresco-enterprise-repo/pom.xml</relativePath>
-        <version>14.26</version>
+        <version>14.27-SNAPSHOT</version>
     </parent>
 
     <properties>
-        <dependency.alfresco-enterprise-repo.version>14.26</dependency.alfresco-enterprise-repo.version>
-        <dependency.alfresco-enterprise-share.version>14.17</dependency.alfresco-enterprise-share.version>
+        <dependency.alfresco-enterprise-repo.version>14.27-SNAPSHOT</dependency.alfresco-enterprise-repo.version>
+        <dependency.alfresco-enterprise-share.version>14.18-SNAPSHOT</dependency.alfresco-enterprise-share.version>
```

Comment:
These all need changing back to release versions before this can be merged.

Reply:
yes, I should rebase before merging: at that point, the build will be red (I guess): these versions are set to that values to be bound to enterprise repo branch